.github/workflows/build.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           mv target/release/symsorter build/symsorter-Linux-x86_64
           mv target/release/symbolicli build/symbolicli-Linux-x86_64
 
-      - uses: actions/upload-artifact@v4.3.1
+      - uses: actions/upload-artifact@v3
```

Comment:
Looks like this was v3.1.1 [here](https://github.com/getsentry/symbolicator/pull/1366/files), let's keep it the same version as before?

Reply:
Done.